File: Python/car.py
from account import Account
from uberX import UberX
from uberPool import UberPool
from uberVan import UberVan 
from uberBlack import UberBlack

class Car:
    id        = int
    license   = str
    driver    = Account("","")
    passenger = int

    def __init__(self, license, driver):
        self.license = license
        self.driver = driver

File: Java/Car.java
class Car {
    private Integer id;
    private String license;
    private Account driver;
    private Integer passenger;

    public Car(String license, Account driver){
        this.license = license;
        this.driver = driver;
    }

    void printDataCar(){
        if(passenger != null){
            System.out.println("License is: " + license + " Driver is: " + driver.name + " numero de pasajeros: " + passenger);
        }
    }

    public Integer getPassenger(){
        return passenger;
    }

    public void setPassenger(Integer passenger){
        if(passenger == 4){
            this.passenger = passenger;
        } else{
            System.out.println("debe tener 4 pasajeros");
        }   
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getLicense() {
        return license;
    }

    public void setLicense(String license) {
        this.license = license;
    }

    public Account getDriver() {
        return driver;
    }

    public void setDriver(Account driver) {
        this.driver = driver;
    }

    


}